### ISSUE TITLE
Troubles with genome-attributes-viewer in the new Docker image

### ISSUE THREAD

**Issue by ggonnella** [ggonnella/fastsubtrees#18]
@vinisalazar wrote:

> I rebuilt the Docker image today after the updates. I still can successfully run the benchmarks from the Docker container, but I was having trouble with the genome-attributes-viewer app, it started, I ran a couple of queries, and then it crashed with this traceback:

```
 root@d8798c592652:/fastsubtrees/docker# ./start-example-app
Dash is running on http://0.0.0.0:8050/

 * Serving Flask app 'start' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
./start-example-app: line 15:   823 Killed                  ./start.py
root@d8798c592652:/fastsubtrees/docker# Traceback (most recent call last):
  File "/fastsubtrees/genomes-attributes-viewer/start.py", line 171, in <module>
    app.run_server(debug=True, host='0.0.0.0')
  File "/usr/local/lib/python3.10/dist-packages/dash/dash.py", line 2033, in run_server
    self.server.run(host=host, port=port, debug=debug, **flask_run_options)
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 920, in run
    run_simple(t.cast(str, host), port, self, **options)
  File "/usr/local/lib/python3.10/dist-packages/werkzeug/serving.py", line 1000, in run_simple
    _rwr(
  File "/usr/local/lib/python3.10/dist-packages/werkzeug/_reloader.py", line 418, in run_with_reloader
    ensure_echo_on()
  File "/usr/local/lib/python3.10/dist-packages/werkzeug/_reloader.py", line 398, in ensure_echo_on
    termios.tcsetattr(sys.stdin, termios.TCSANOW, attributes)
termios.error: (5, 'Input/output error')
```

**Comment by ggonnella** [comment]
I could not yet reproduce this.
I fixed the version of the Werkzeug package, because I had other issues with newer versions.
I will try again to change its version, to see if using a newer version helps.

**Comment by ggonnella** [comment]
Oh no... it was a problem of the automatisms... I never noticed the issue in my system, because the files existed already and were not re-created.

I fixed it in the latest commit eef44fd2b9559c83eb67eb36d634d310a8fc9079 and uploaded to Pypi a version 1.3 of the genome_attribute_viewer package with the fix.

**Comment by KonradHoeffner** [comment]
Can confirm, it works now for me!
